cogs/commands/misc.py

```diff
@@ -143,6 +143,17 @@ async def chat(self, ctx: Context, message: str = None, gpt4: bool = False):
     async def gpt4(self, ctx: Context, message: str):
         await self.chat(ctx, message, True)
 
+    @commands.hybrid_command()
+    async def translate(
+        self, ctx: Context, source: str, target: str, message: clean_content
+    ):
+        translated = GoogleTranslator(source=source, target=target).translate(message)
+        await ctx.send(translated)
+
+    @commands.hybrid_command()
+    async def joeltech(self, ctx: Context):
```

Comment:
tbh im not entirely sure if we need this given :joel_tech: can already be written

Reply:
the command should be Joel not Joel tech for this to be effective